feedback-module/src/components/common/Module.js

```diff
@@ -40,7 +92,50 @@ function Module() {
       className={MODULE_CONTAINER_STYLE}
     >
       <Header />
-      {screen}
+      {screen.titleInverse && (
+        <Grid className={`bg-primary ${SCREEN_CONTAINER_STYLE}`}>
+          <p
+            className={`${H1_WHITE_STYLE}`}
+            dangerouslySetInnerHTML={{ __html: screen.titleInverse }}
+          ></p>
+        </Grid>
+      )}
+      {screen.buttons && (
+        <Grid className={SCREEN_CONTAINER_STYLE}>
+          {screen.title && <p className={H1_STYLE}>{screen.title}</p>}
```

Comment:
Can we take this header text and plainText outside the check if screen.buttons is not null (for example put it between line 102 and 103) because there is a possibility in the future of having a non inverse title on top of the module even if there are no buttons on that particular screen.